inst/tests/tests.Rraw

```diff
@@ -9752,11 +9673,6 @@ test(1685, grep("dtu", capture.output(foo())), 7L)
 test(1686.1, uniqueN(1L), 1L)
```

Comment:
iiuc, if this 1686.1 was changed to 1685.1, the 4 lines just below (1681.1 and 1686.2) wouldn't need to change or move?

Reply:
seems right. can re-run the dup check to be sure.